### PR TITLE
Booking model - Please review

### PR DESCRIPTION
**Pull request by gguillaume777**
Screen Shot 2021-02-10 at 11 00 35 AM
Screen Shot 2021-02-10 at 11 00 27 AM
Screen Shot 2021-02-10 at 11 00 18 AM

### CODE REVIEW

File: db/migrate/20210209210154_create_bookings.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateBookings < ActiveRecord::Migration[6.1]
+  def change
+    create_table :bookings do |t|
+      t.string :date
```
Comment (cuneyter) [inline db/migrate/20210209210154_create_bookings.rb:4]: I think in row 4 data type should be t.date. 

i have looked at below tutorials. Please have a look at let me know what you think? 
https://www.tutorialspoint.com/ruby-on-rails/rails-migrations.htm
https://uday.net/rails-datatypes-for-ActiveRecord

File: db/migrate/20210209210154_create_bookings.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+      t.string :text
```
Comment (cuneyter) [inline db/migrate/20210209210154_create_bookings.rb:5]: I think row 5 should be t.date and column name should be :end_date

File: db/migrate/20210209210154_create_bookings.rb (hunk 1 above)
Comment (gguillaume777) [reply db/migrate/20210209210154_create_bookings.rb:4]: Hey !

great point, I then think we should have something like:

t.date :start_date
t.date :end_date

and for the column name:
t:date "start_date"
t:date "end_date"

agree?

File: db/migrate/20210209210154_create_bookings.rb (hunk 2 above)
Comment (gguillaume777) [reply db/migrate/20210209210154_create_bookings.rb:5]: yessss perfect

File: db/migrate/20210209210154_create_bookings.rb (hunk 1 above)
Comment (cuneyter) [reply db/migrate/20210209210154_create_bookings.rb:4]: I think we need need to run the rails g model booking again and the migrations should be as follows:

t.date :start_date
t.date :end_date

**Comment by cuneyter** [comment]
as agreed, i am closing this pull request for now and we need to rerun the model generator. please let me know if you need any help

**Review by cuneyter — APPROVED** [review]
Thank you for making these changes. Looks good to me now

**Comment by cuneyter** [comment]
Looks great. thank you

**Pull request merged**